src/daemon/collector/overload/waist-requests.js

```diff
@@ -10,18 +10,19 @@
  * }}
  */
 function getWaistRequestVariablesFromSponsoredPost(node) {
-  const variables = {};
+  let variables = {};
```

Comment:
You could leave this as a `const` and later on just assign properties, i.e. 

`variables.adId = ad_id`